assets/scss/01-atoms/_fixed-feedback-button.scss

```diff
@@ -0,0 +1,28 @@
+.ma__fixed-feedback-button {
+  position: fixed;
+  width: 40px;
+  bottom: 0;
+  right: 0;
+  height: 300px;
+  pointer-events: none;
+  
+  a {
+    display: block;
+    position: absolute;
+    top: 115px;
+    left: -40px;
+    font-size: 1.5rem;
+    font-weight: 500;
+    letter-spacing: 0.15rem;
+    color: $c-accessible-feedback;
+    background-color: $c-white;
+    border-style: solid;
+    border-width: 4px 4px 0 4px;
```

Comment:
Technically, the last `4px` is not needed, but likely not worth addressing at this point. OCD makes me point it out :)